### PR TITLE
Add openscad and sca2d support

### PR DESCRIPTION
**Pull request by n8henrie**

### CODE REVIEW

**Comment by n8henrie** [comment]
Hmmm, sorry about the failing lings. Having a hard time running the docker tests on my M1 mac.

Revision soon.

**Comment by n8henrie** [comment]
Man, I don't know what's going on with the tests, but `./run-tests` is hanging for ages on every run and rarely completing. Rebooted, tried deleting and redownloading the images. Wonder if it has to do with ARM / having to `export DOCKER_DEFAULT_PLATFORM=linux/amd64`.

Seems like letting CI do the feedback may end up being quicker.

**Comment by n8henrie** [comment]
@w0rp -- I think this is just missing a review and otherwise could be merged.

**Comment by hsanson** [comment]
Tests are failing in Windows.  Make sure to use ALE's path utils to ensure things work properly on all platforms.

**Comment by n8henrie** [comment]
> Tests are failing in Windows.

Thanks for the feedback, I hadn't realized CI wasn't running on windows.

Tried with `ale#path#Simplify`, hopefully that does the trick.

**Comment by n8henrie** [comment]
Gah, I didn't scroll enough to see the appveyor failure (nor that it was an incomplete log on the site and needed to be downloaded). Should be able to get this passing now.

**Comment by n8henrie** [comment]
Looks like windows is finally passing so I've rebased and squashed.

File: supported-tools.md [hunk 1]
```diff
@@ -405,6 +405,8 @@ formatting.
   * [ibm_validator](https://github.com/IBM/openapi-validator)
   * [prettier](https://github.com/prettier/prettier)
   * [yamllint](https://yamllint.readthedocs.io/)
+* OpenSCAD
+  * [SCA2D](https://gitlab.com/bath_open_instrumentation_group/sca2d)
```
Comment (hsanson) [inline supported-tools.md:409]: Since this tool works with files on disk we need to add the :floppy_disk: emoji.

**Review by hsanson — CHANGES_REQUESTED** [review]

File: supported-tools.md (hunk 1 above)
Comment (n8henrie) [reply supported-tools.md]: No problem, sorry I missed that.

**Pull request merged**